lib/vsc/utils/fancylogger.py

```diff
@@ -450,18 +450,24 @@ def logToFile(filename, enable=True, filehandler=None, name=None, max_bytes=MAX_
 
     if you want to disable logging to file, pass the earlier obtained filehandler
     """
-    handleropts = {'filename': filename,
-                   'mode': 'a',
-                   'maxBytes': max_bytes,
-                   'backupCount': backup_count,
-                   }
-    return _logToSomething(logging.handlers.RotatingFileHandler,
-                           handleropts,
-                           loggeroption='logtofile_%s' % filename,
-                           name=name,
-                           enable=enable,
-                           handler=filehandler,
-                           )
+    handleropts = {
+        'filename': filename,
+        'mode': 'a',
+        'maxBytes': max_bytes,
+        'backupCount': backup_count,
+    }
+    # logging to a file is going to create the file later on, so let's try to be helpfull and create the path if needed
+    directory = os.path.dirname(filename)
+    if not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
it was already raising these errors, when trying to open the file, this step just makes it less likely to have it raised, should I also catch the os error that get's thrown when trying to write to a file that can't be created?  I explicitly didn't intend to catch any errors, just add this as a convinience so I don't have to always create the dir before calling logToFile, but logging to a file can always fail, and I don't think we should catch each individual case. E.g. When the user doesn't have permission to write tot he file, or the disk is full ( or over quota) or the filesystem is mounted read only, etc..
Each of these errors would have a different exception raised, and need a different action from the user. I don't want to go around and catch them all, to just raise them with a clearer message? 
Then what happens when a new type of filesystem is introduced that can throw new errors,  I need to keep the list of things that can fail up to date here?